pkg/crc/machine/kubeconfig_test.go

```diff
@@ -160,26 +160,27 @@ func Test_addContext(t *testing.T) {
 		username   string
 		context    string
 		token      string
+		namespace  string
 	}
 
 	tests := []struct {
 		in       input
 		expected string
 	}{
 		{
-			input{"https://abcdd.api.com", "foo", "foo@abcdd", "secretToken"},
+			input{"https://abcdd.api.com", "foo", "foo@abcdd", "secretToken", "kube-system"},
 			"foo/abcdd-api-com",
 		},
 		{
-			input{"https://api.crc.testing:6443", "kubeadmin", "kubeadm", "secretToken"},
+			input{"https://api.crc.testing:6443", "kubeadmin", "kubeadm", "secretToken", "default"},
 			"kubeadmin/api-crc-testing:6443",
 		},
```

Comment:
@anjannath we should add one more test case which have empty string as namespace which make sure the context is added to kubeconfig but without namespace.

Reply:
updated the test